app/views/support/ABTest.scala

```diff
@@ -66,12 +73,26 @@ trait TestTrait {
      cdf.zip(filteredVariantList)
    }
 
-    variantsByCountry.map { case (country, variants) => country -> addRanges(variants)}
+    variantsByCountry.map { case (country, variants) => country -> addRanges(variants) }
   }
 }
 
+object ReducedCheckoutTest extends TestTrait {
+  def name = "ReducedCheckoutTest"
+
+  def slug = "2sc"
+
+  def variants = NonEmptyList(
+    makeVariant("control", "c", 10, Some(BooleanVariantData(false))),
+    makeVariant("test", "t", 1, Some(BooleanVariantData(true)))
```

Comment:
Because you're only providing `true` and `false` as the variant data, you can just use `None` here, and avoid having to specify the json formatter. On the client you can just check which variant you're in rather than reading the data.